### PR TITLE
item.display bugfixes and moved item open/close/locked to properties

### PR DESCRIPTION
**Pull request by marcelomoreli**
Fixed a lot of bugs with item.display and moved item closed/locked/locked by to Properties.
Also modified open/close/lock/unlock commands so that they are a single command.

### CODE REVIEW

**Comment by shawncplus** [comment]
@marcelomoreli This branch has conflicts since the merge of the `use` fix

**Comment by marcelomoreli** [comment]
Done

**Comment by marcelomoreli** [comment]
Potentially done as well?

File: bundles/ranvier-commands/commands/open.js [hunk 1]
```diff
@@ -18,102 +21,204 @@ module.exports = (srcPath, bundlePath) => {
...
+              }
+
+              return B.sayAt(player, "The door is locked and you don't have the key.");
+            }
+            if (door.closed) {
+              B.sayAt(player, "The door swings opened.");
+              return doorRoom.openDoor(targetRoom);
             }
           }
+          if (action == 'close') {
```
Comment (shawncplus) [inline bundles/ranvier-commands/commands/open.js]: This should be a switch

File: bundles/ranvier-commands/commands/use.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ module.exports = (srcPath, bundlePath) => {
   const Broadcast = require(srcPath + 'Broadcast');
   const Logger = require(srcPath + 'Logger');
   const { CommandParser } = require(srcPath + 'CommandParser');
+  const ItemUtil = require(bundlePath + 'ranvier-lib/lib/ItemUtil');
```
Comment (shawncplus) [inline bundles/ranvier-commands/commands/use.js]: This can be removed

File: bundles/ranvier-commands/commands/wear.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ module.exports = (srcPath, bundlePath) => {
   const ItemUtil = require(bundlePath + 'ranvier-lib/lib/ItemUtil');
   const Logger = require(srcPath + 'Logger');
   const say = Broadcast.sayAt;
+  const ItemUtil = require(bundlePath + 'ranvier-lib/lib/ItemUtil');
```
Comment (shawncplus) [inline bundles/ranvier-commands/commands/wear.js]: Same here

File: bundles/ranvier-commands/commands/open.js [hunk 4]
```diff
@@ -18,102 +21,204 @@ module.exports = (srcPath, bundlePath) => {
...
+                  B.sayAt(player, `*Click* You unlock the door with ${ItemUtil.display(playerKey)} and open it.`);
+                  doorRoom.unlockDoor(targetRoom);
+                  return doorRoom.openDoor(targetRoom);
+                }
+              }
+
+              return B.sayAt(player, "The door is locked and you don't have the key.");
+            }
+            if (door.closed) {
+              B.sayAt(player, "The door swings opened.");
```
Comment (seanohue) [inline bundles/ranvier-commands/commands/open.js]: `"The door swings open."` is more idiomatic English

**Comment by marcelomoreli** [comment]
Suggestions applied

**Pull request merged**